tests/sanity_test.go

```diff
@@ -0,0 +1,41 @@
+package tests_test
+
+import (
+	"flag"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/kubevirt/pkg/api/v1"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/tests"
```

Comment:
why do we need to import the tests?

Reply:
it has `utils` with a lot of useful methods, like create VM with registry disk.

Reply:
looking at it, i bet it will split into multiple files in the future :)
but seems useful